vm/systemSmartContracts/proto/governance.proto

```diff
@@ -41,17 +46,25 @@ message GovernanceConfig {
     bytes ProposalFee      = 5 [(gogoproto.jsontag) = "ProposalFee", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
 }
 
-message VoterData {
-    bytes  Address  = 1 [(gogoproto.jsontag) = "Address"];
-    int32 NumNodes = 2 [(gogoproto.jsontag) = "NumNodes"];
+message GovernanceConfigV2 {
+    bytes MinQuorum        = 1 [(gogoproto.jsontag) = "MinQuorum", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
```

Comment:
What are the MinQuorum, MinPassThreshold, MinVetoThreshold measured in? Number of votes, voting power, voting balance? Voting balance seems the easiest one. The number of votes can be faked by splitting the balance into multiple accounts and this way also faking the "voting power".

Reply:
Voting power is the minQuorum. If you split you have to delegate from all those accounts and do a lot of transactions. You are also limited to minimum 1 eGLD per send.

Reply:
ok